skrebate/relieff.py

```diff
@@ -113,6 +113,42 @@ def find_neighbors_binary(
     return np.array(nn_list)
 
 
+# interestingly, this is sometimes faster without numba
+def compute_score_binary(
```

Comment:
This is the core of the performance improvements for score computation - it expresses the computation as a sequence of numpy matrix operations.